egograph/dataset_catalog/canonical.py

```
@@ -0,0 +1,103 @@
+"""Parquet canonical type conversion helpers.
+
+Pipelines（pyarrow で出力検証）と Backend（DuckDB で fixture 検証）が
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)